content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -39,17 +39,17 @@ One of the [recommended labels](/docs/concepts/overview/working-with-objects/com
 
 Example: `app.kubernetes.io/instance=mysql-abcxzy`
 
-Used on: All Objects
+Used on: All Objects (typically used on [Workload Resources](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/)).
 
-A unique name identifying the instance of an application.
+A unique name identifying the instance of an application. If you want to declare what application this is part of but not in a unique way, use [app.kubernetes.io/name](##app-kubernetes-io-name).
 
 One of the [recommended labels](/docs/concepts/overview/working-with-objects/common-labels/#labels).
 
 ### app.kubernetes.io/managed-by
 
 Example: `app.kubernetes.io/managed-by=helm`
 
-Used on: All Objects
+Used on: All Objects (typically used on [Workload Resources](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/)).
```

Comment:
Remove [caps](https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects), use [relative links](https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects).
```suggestion
Used on: All Objects (typically used on [workload resources](/docs/reference/kubernetes-api/workload-resources/)).
```